lib/tasks/deploy.rake

```diff
@@ -98,9 +98,11 @@ namespace :deploy do
   # - remote: list
   # - remote: stop
   task :stop do
-    on_each_docker_host do |server|
-      stop_containers(server, defined_service, fetch(:stop_timeout, 30))
+    threads = on_each_docker_host.map do |server|
```

Comment:
Nope, not sure at all -- I'm really not a ruby dev. I just assumed that since you can `do |x|` it then you can `map` it as well.